pkg/sink/sink.go

```diff
@@ -102,12 +102,15 @@ func (r Sink) HandleEvent(response http.ResponseWriter, request *http.Request) {
 		go func(t triggersv1.EventListenerTrigger) {
 			finalPayload := event
 			if interceptor != nil {
-				payload, err := interceptor.ExecuteTrigger(event, request, &t, eventID)
+				payload, header, err := interceptor.ExecuteTrigger(event, request, &t, eventID)
 				if err != nil {
 					log.Error(err)
 					result <- http.StatusAccepted
 					return
 				}
+				if header != nil {
```

Comment:
Is a response with no headers treated as nil or an empty map? 

We might just want to explicitly return req.Headers everywhere explicitly.

Reply:
Done. Now I return an empty map instead of nil.